config/rbac/service_account.yaml

```diff
@@ -0,0 +1,6 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: sa
+  namespace: system
```

Comment:
drop namespace from here so it can be applied dynamically as part of installation, vs hard-coded to this ns, to fix the failure @adamgoossens

Reply:
Thanks :) I just bumped into that my self. Will adjust it now...